### PR TITLE
Block external font-face set

### PR DESCRIPTION
**Pull request by weizman**
Address #48

* strongly attach unselectable to a non-existing font-face so outer ones won't stick
* change unselectable to hardened (more relevant)

### CODE REVIEW

File: packages/core/src/element.mjs [hunk 1]
```diff
@@ -36,8 +36,10 @@ function creator(style, tag, text = '') {
 
 const invoker = creator => () => creator();
 
-// an element that is hard to find/select
-export const unselectable = invoker(creator({
+// an element that it should be hard to find/select/leak from/etc
+export const hardened = invoker(creator({
+    // decide on an unguessable font-family (non-existing) so an external one cannot be applied
+    'font-family': rand(20),
```
Comment (legobeat) [inline packages/core/src/element.mjs:42]: Why not go even higher than `20`?

**Comment by legobeat** [comment]
In a way, does this not serve to aid the vector it is addressing - by forcing defaults, a dictionary of common or known defaults can serve to also predict the height...?

File: packages/core/src/element.mjs (hunk 1 above)
Comment (weizman) [reply packages/core/src/element.mjs:42]: What number would be sufficient?

**Comment by weizman** [comment]
> In a way, does this not serve to aid the vector it is addressing - by forcing defaults, a dictionary of common or known defaults can serve to also predict the height...?

The problem was that without this setting, it was possible for attackers to change the height of specific chars (as opposed to all of them) which would provide feedback to the attacker on which chars exist and which don't (and in what order).

With this change, all of LavaDome's chars obey the same height restriction, so telling their height from outside is possible, but it isn't useful to tell them apart, so in practice attackers gain no information from such tactic.

**Review by legobeat — APPROVED** [review]

**Pull request merged**